src/style.css

```diff
@@ -0,0 +1,26 @@
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
```

Comment:
the last star shouldn't have margin-right

Reply:
Hi:) I need this task solution to be counted in the BEM module on the Mate Academy platform. Should i  make a new pull request?